### PR TITLE
release/0.15.0

### PR DESCRIPTION
**Pull request by vanbasten17**
# Description
- [x] Added changelog
- [x] Update documentation with NLU updates
- [x] Bump all versions to rc
- [x] Test all templates
- [x] Bump all versions to final version

### CODE REVIEW

**Review by annehubtype — APPROVED** [review]

File: doc-botonic/docs/project-structure.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ Actions are where you define the behavior of your bot. You can add an Action by
 
 You can go a long way capturing user inputs using regular expressions, but it obviously has its limitations. As you find yourself adding more and more functionality to your bot, you get to a point where you need Natural Language Understanding (NLU) capabilities.
 
-NLU lets you capture user inputs by `intent` instead of parsing its raw text. An intent represents all the different ways users can express a unit of meaning that is valid for your bot. For example, you can group the sentences "What's the weather in California like?" and "Do you know if it's sunny today in California?" to the intent `GetWeather` and the parameter `city=California`. You can then map that intent to an action using a route.
+NLU lets you capture user inputs by `intent` instead of parsing its raw text. An intent represents all the different ways users can express a unit of meaning that is valid for your bot.<!-- For example, you can group the sentences "What's the weather in California like?" and "Do you know if it's sunny today in California?" to the intent `GetWeather` and the parameter `city=California`. --> You can then map that intent to an action using a route.
```
Comment (asastre) [inline doc-botonic/docs/project-structure.md:51]: Why not removing these commented lines?

**Review by asastre — CHANGES_REQUESTED** [review]
Just 2 things to review before approving

File: doc-botonic/docs/project-structure.md (hunk 1 above)
Comment (vanbasten17) [reply doc-botonic/docs/project-structure.md:51]: atm, we want to preserve them as it's very likely we can take profit of them in next days, once we have the entities engine working

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,69 @@ All notable changes to Botonic will be documented in this file.
...
+  - New botonic.io [website](https://github.com/hubtype/botonic/pull/1014).
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - New webchat features:
+
+    - Added new [`target`](https://github.com/hubtype/botonic/pull/970) prop to `Button` component to define the behavior to open links.
+    - Added new [`WhatsappTemplate`](https://github.com/hubtype/botonic/pull/972) component for Webchat.
+    - New [`getBotonicApp()`](https://github.com/hubtype/botonic/pull/1018) to access `Botonic` methods.
+    - Added a new webchat setting [shadowDOM](https://github.com/hubtype/botonic/pull/1020) that indicates whether the whole webchat component should be encapsulated in a shadow DOM node.
```
[the remark this answers is not in the document]
Comment (vanbasten17) [reply CHANGELOG.md]: corrected 👍

**Review by asastre — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,68 @@ All notable changes to Botonic will be documented in this file.
...
+    - Added new [`WhatsappTemplate`](https://github.com/hubtype/botonic/pull/972) component for Webchat.
+    - New [`getBotonicApp()`](https://github.com/hubtype/botonic/pull/1018) to access `Botonic` methods.
+    - Added a new webchat setting [`storageKey`](https://github.com/hubtype/botonic/pull/1019) that indicates the key name to use in order to store the webchat state in `localStorage` (or `sessionStorage`).
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+  - [Improvements](https://github.com/hubtype/botonic/pull/985) in Content.validate of text & shortText fields
+    - Validate text.text also if content has no keywords (it could be accessed through a button)
+    - Before, when shortText was empty, the contentful driver set it with the value of the name field (to ensure buttons did not show blank texts). Now this is managed by the Button model, so that MessageContent.validate() can report it.
+    - New [CMS method](https://github.com/hubtype/botonic/pull/1002) to deliver content from `id` when `type` is unknown.
```
Comment (dpinol) [inline CHANGELOG.md:35]: content by `id` when the content type is unknowm

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,68 @@ All notable changes to Botonic will be documented in this file.
...
+    - Improved [ContentsValidator](https://github.com/hubtype/botonic/pull/1001). Better logs through which you can get a list of errors.
```
Comment (dpinol) [inline CHANGELOG.md:36]: replace "Better logs through which you can get a list of errors." with "Now it's possible to report the detected errors into a callback".

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +11,68 @@ All notable changes to Botonic will be documented in this file.
...
+  - [Fixed](https://github.com/hubtype/botonic/pull/1015) imports in templates which were broken after 0.14.0 change in entry files.
+  - Added validation of templates and examples in CI
+
+- [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - `session` properties now are [merged and updated](https://github.com/hubtype/botonic/pull/973) correctly.
+  - Avoid [customMessage to crash](https://github.com/hubtype/botonic/pull/1000) if they have bad children. Some bots may store no-react objects in children.
+  - [Fixed](https://github.com/hubtype/botonic/pull/1017/) error appearing in some tests: `Received true for a non-boolean attribute markdown`. Caused by styled-components.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
```
Comment (dpinol) [inline CHANGELOG.md:73]: you can remove this point. It's very minor

File: packages/botonic-plugin-nlu/README.md [hunk 6]
```diff
@@ -115,6 +115,28 @@ export const routes = [
...
+|   Hindi    |      hi       | 10k-fasttext |    300     | **[10k-fasttext-300d-hi](https://s3-eu-west-1.amazonaws.com/word-embeddings.hubtype.com/10k-fasttext-300d-hi.db)** |
+| Indonesian |      id       | 10k-fasttext |    300     | **[10k-fasttext-300d-id](https://s3-eu-west-1.amazonaws.com/word-embeddings.hubtype.com/10k-fasttext-300d-id.db)** |
+|  Russian   |      ru       | 10k-fasttext |    300     | **[10k-fasttext-300d-ru](https://s3-eu-west-1.amazonaws.com/word-embeddings.hubtype.com/10k-fasttext-300d-ru.db)** |
+|  Turkish   |      tr       | 10k-fasttext |    300     | **[10k-fasttext-300d-tr](https://s3-eu-west-1.amazonaws.com/word-embeddings.hubtype.com/10k-fasttext-300d-tr.db)** |
+
+_\*10k-fasttext word embeddings contain the 10k most commons words of the language._
+
+**Note:** All the word embeddings are stored by default in `~/.botonic/word-embeddings/`. If you encounter any issues when automatically downloading word embeddings, you can download them manually and store it in the mentioned directory.
```
Comment (dpinol) [inline packages/botonic-plugin-nlu/README.md:142]: "store it" -> "store them"

**Review by dpinol — CHANGES_REQUESTED** [review]

**Pull request merged**